indico/modules/categories/templates/management/settings.html

```diff
@@ -0,0 +1,62 @@
+{% from 'forms/_form.html' import form_header, form_rows, form_fieldset, form_footer %}
+
+{% extends 'categories/management/base.html' %}
+
+{% block title %}
+    {% trans %}Settings{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    {{ form_header(form) }}
+        {{ form_rows(form, fields=('title', 'description', 'timezone', 'lecture_theme',
+                                   'meeting_theme', 'visibility')) }}
+        {% call form_fieldset(_('Event Header')) %}
+            {{ form_rows(form, fields=('event_message_mode', 'event_message')) }}
```

Comment:
what about storing the name of these fields in an attribute on the form and then using that attribute here (and in `skip` on the previous `form_rows` call)?